scripts/settings.js

```diff
@@ -612,4 +612,4 @@ function setRemoveBackgroundCookie() {
 
 function exportToJson() {
   window.location.href = "endpoints/subscriptions/export.php";
-}
\ No newline at end of file
+}

```

Comment:
On this file:

`switchTheme` `setShowMonthlyPriceCookie` `setConvertCurrencyCookie` `setRemoveBackgroundCookie` will set the cookie to `/`

` path=/` should be removed.